app/presenters/tree_builder.rb

```diff
@@ -257,9 +257,15 @@ def x_build_node(object, pid, options)
     node = x_build_single_node(object, pid, options)
 
     # Process the node's children
+    load_children = if object.kind_of?(Struct)
```

Comment:
Sorry, that was definitely not my intention :)

However, all the trees dealing with `Vm`, etc. don't expect a hash here either, which leads me to think your code should not go through here. Any chance there's something else broken?